roles/confluent.common/tasks/redhat.yml

```diff
@@ -1,7 +1,7 @@
 - name: Install Java
   yum:
     name: "{{redhat_java_package_name}}"
-    state: latest
+    state: present
```

Comment:
This appears to be a correction for a different issue, can you confirm this?  As its not mentioned in the description.

Reply:
sorry this is not related, but I realized we dont really have a requirement that java 1.8 is the latest, but present. Its up to the users when they want to update their java